examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -21,6 +21,9 @@ substitutions:
 :author: Nathaniel Forde
 :::
 
+:::{include} ../extra_installs.md
```

Comment:
I noticed you had set the substitutions metadata but this include was missing. It should now add some auto instructions on installing lifelines.

Reply:
Oh, cool. Yeah, i forgot this,